python compilation failed!

###output:


###error:
Traceback (most recent call last):
  File "/Users/zhaozhaozhaozhao/Unambigious_prompt/exp/output_tb_gen_tb_20250319/47/pychecker_4.py", line 66, in <module>
    outputs = check_output(stimulus_list)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/zhaozhaozhaozhao/Unambigious_prompt/exp/output_tb_gen_tb_20250319/47/pychecker_4.py", line 49, in check_output
    tb_outputs.append(dut.load(stimulus))
                      ^^^^^^^^^^^^^^^^^^
  File "/Users/zhaozhaozhaozhao/Unambigious_prompt/exp/output_tb_gen_tb_20250319/47/pychecker_4.py", line 21, in load
    a = BinaryValue(stimulus['a']).integer
        ^^^^^^^^^^^
NameError: name 'BinaryValue' is not defined
